pkg/apis/virt/v1alpha1/types.go

```diff
@@ -114,6 +116,14 @@ type ContainerRootfsVolumeSource struct {
 	Size            resource.Quantity `json:"size"`
 }
 
+type PersistentVolumeClaimSource struct {
+	ClaimName string `json:"claimName"`
+}
+
+type DataVolumeSource struct {
+	Name string `json:"name"`
```

Comment:
Better name it `VolumeName`, since the struct is named `DataVolumeSource` (not `DataVolume`)